files/templates/alertmanager/alertmanager.yaml

```diff
@@ -17,6 +17,12 @@ route:
   receiver: root
 
   routes:
+
+  - receiver: harbor_implementation
+    matchers:
+    - team="harbor"
```

Comment:
Why it's `team="harbor"` ?

Reply:
This is simply for testing alerts which are being implemented by ContainerSolutions. No other internal team should get / own these currently. Does this work to just set harbor?

Reply:
it's kind of misleading configuration, We forward alerts based on Giant Swarm Teams.
So, I prefer if you can use another matcher.. instead of `team="harbor"`

Reply:
Will change it.